### ISSUE TITLE
Persona secondary and tertiary text does not meet default contrast with DarkCustomizations

### ISSUE THREAD

**Issue by paulgildea** [microsoft/fluentui#12844]
### Environment Information

- **Package version(s)**: 7.107.3
- **Browser and OS versions**: Microsoft Edge Beta - Version 81.0.416.60

### Please provide a reproduction of the bug in a codepen:
Issue can be reproduced on the Fluent UI Control Website: https://developer.microsoft.com/en-us/fluentui#/controls/web/persona

Go to the live code example and enable Dark Theme.

![Persona Issue](https://user-images.githubusercontent.com/33229964/80137893-d11a4f00-8558-11ea-832e-eeefdd191a4f.gif)

#### Actual behavior:

Persona secondaryText and tertiaryText does not meet the default contrast ratio

#### Expected behavior:

Persona secondaryText and tertiaryText does meets the default contrast ratio 

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: High - High Visibility

Products/sites affected: Fluent UI Control Website: https://developer.microsoft.com/en-us/fluentui#/controls/web/persona

**Comment by jenniaynlynn** [comment]
Not sure if this is a similar issue or not, but we've also found the persona with initials instances also have contrast issues when they have backgrounds other than white.  For example the persona with initials when used on the O365 NavHeader.  Would love to see a 1px border to create enough contrast between the persona and the background.

![image](https://user-images.githubusercontent.com/44474462/89833906-b8897100-db16-11ea-8c4c-804519f76862.png)
